clusters/build-clusters/vsphere/cert-manager/apiserver-vsphere_certificate.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  labels:
+    aws-project: openshift-ci-infra
+  name: apiserver-vsphere-tls
+  namespace: openshift-config
+spec:
+  secretName: apiserver-vsphere-tls
+  issuerRef:
+    name: cert-issuer-aws
+    kind: ClusterIssuer
+  dnsNames:
+  - 'api.build01-us-west-2.vmc.ci.openshift.org'
```

Comment:
If requested, we could give the ownership of `build01-us-west-2.vmc.ci.openshift.org` to `vsphere` team.